### PR TITLE
Addressing requested changes

### PR DESCRIPTION
**Pull request by somindatommy**
Fix comments in https://github.com/wso2-extensions/identity-inbound-provisioning-scim2/pull/167

### CODE REVIEW

File: components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/DAO/GroupDAO.java [hunk 1]
```diff
@@ -442,8 +443,9 @@ public void updateRoleName(int tenantId, String oldRoleName, String newRoleName)
                     while (rSet.next()) {
                         String roleName = rSet.getString(1);
                         if (StringUtils.isNotEmpty(roleName)) {
-                                // Remove the primary domain name from roleNames.
-                                roleList.add(removePrimaryDomainName(roleName));
+
```
Comment (madurangasiriwardena) [inline components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/DAO/GroupDAO.java]: No need to have a new line.

File: components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/DAO/GroupDAO.java [hunk 2]
```diff
@@ -423,6 +423,7 @@ public void updateRoleName(int tenantId, String oldRoleName, String newRoleName)
 
         // Resolve sql query for filtering.
         if (StringUtils.isNotEmpty(domainName)) {
+
```
Comment (madurangasiriwardena) [inline components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/DAO/GroupDAO.java]: No need to have a new line.

File: components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/DAO/GroupDAO.java (hunk 1 above)
Comment (somindatommy) [reply components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/DAO/GroupDAO.java]: Fixed: 7f95a33846b4f8b080987cf395121938ee0c6ceb

File: components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/DAO/GroupDAO.java (hunk 2 above)
Comment (somindatommy) [reply components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/DAO/GroupDAO.java]: Fixed: 7f95a33846b4f8b080987cf395121938ee0c6ceb

**Pull request merged**